airflow/providers/google/cloud/hooks/dataprep.py

```diff
@@ -205,3 +219,74 @@ def _raise_for_status(self, response: requests.models.Response) -> None:
         except HTTPError:
             self.log.error(response.json().get("exception"))
             raise
+
+    @retry(stop=stop_after_attempt(5), wait=wait_exponential(multiplier=1, max=10))
+    def create_imported_dataset(self, *, body_request: dict) -> dict:
+        """
+        Creates imported dataset.
+
+        :param body_request: Body of the POST request to be sent.
+            For more details check https://clouddataprep.com/documentation/api#operation/createImportedDataset
+        """
+        endpoint = "/v4/importedDatasets"
```

Comment:
If we are left with v4 lets define it globally rather than in each function

Reply:
Good point, thanks! I added a parameter to the hook `api_version: str = "v4"`, so it's defined in a single place now.